### PR TITLE
Test: Add tests for WittyPixelsTokenVault contract 

### PR DESCRIPTION
**Pull request by guidiaz**

### CODE REVIEW

File: contracts/WittyPixelsTokenVault.sol [hunk 1]
```diff
@@ -175,32 +193,30 @@ contract WittyPixelsTokenVault
...
+        ) {
+            // upon first redemption from playerAddress, add it to the author's list
             __storage.authors.push(_deeds.playerAddress);
-        } 
-        __storage.legacyPixels[_deeds.playerAddress] = _currentPixels + _deeds.playerPixels;
+            __storage.redeemed[_deeds.playerAddress] = true;
+        }
+        if (_deeds.playerPixels > 0) {
+            __storage.legacyPixels[_deeds.playerAddress] = _currentPixels + _deeds.playerPixels;    
+        }
```
Comment (aesedepece) [inline contracts/WittyPixelsTokenVault.sol:215]: I'm  wondering: before this point in which the amount of `playerPixels` are disclosed to the contract, how would it know about the `legacyPixels` it had? How's that known in advance? I must be missing something :thinking:

Comment (guidiaz) [reply contracts/WittyPixelsTokenVault.sol:215]: - Upon fractionalization, the vault contract only knows about total number of pixels in the canvas.
- The "legacyPixels" of any given `playerAddress` will be revealed when valid redemption/s from such `playerAddress` is/are executed.

Comment (guidiaz) [reply contracts/WittyPixelsTokenVault.sol:215]: In other words, only players that decide to redeem ownership will be included in the authors list returns by `getAuthorsRange(,)`.

Comment (aesedepece) [reply contracts/WittyPixelsTokenVault.sol:215]: Then I don't understand why we are incrementing similarly to `__storage.legacyPixels[_deeds.playerAddress] += _deeds.playerPixels` here, if `__storage.legacyPixels[_deeds.playerAddress]` is known to be zero at this point :sweat_smile:

Comment (guidiaz) [reply contracts/WittyPixelsTokenVault.sol:215]: We pretend to optimize all possible paths:
- currentPixels == 0 OR currentPixels != 0
- deeds.playerPixels == 0 OR deeds.playerPixels > 0
- first redemption from `playerAddress` OR not first redemption from `playerAddress

**Review by aesedepece — APPROVED** [review]

**Pull request merged**